### ISSUE TITLE
A11y_Visual Studio Code_Welcome Screen_Info and Relationships:Voice Over is not announcing the label Names(Start, Recent, Getting Started)in Welcome page..

### ISSUE THREAD

**Issue by Latha2564** [microsoft/vscode#131906]
GitHubTags:#A11y_VSCode;#A11ySev2;#A11yTCS;#A11yMAS;#DesktopApp;#Visual Studio Code Client;#WCAG1.3.1;#Infoand Relationships;#MAC;#AINotInScope;#VoiceOver;#Benchmark-VisualStudioCode-MAC-Aug2021

### Environment Details:​
Application Name: Visual Studio Code
Standard Violated: Info and Relationships

### Repro Steps:​
1.Open Visual Studio Code editor
2.Tab till Welcome tab and press enter
3.Verify whether the Voice Over is reading the label Names like(Start, Recent, Getting Started)in Welcome page or not.

### Actual Result:
Voice Over is not reading the label Names like(Start, Recent, Getting Started)in Welcome page.

### Expected Result: ​
Voiceover is should announce the Label Names like(Start, Recent, Getting Started)in Welcome page.

### User Impact:
Screen Reader users will find difficult in finding which elements are under which label if the Voiceover is not announcing the Label Names like like(Start, Recent, Getting Started)in Welcome page.

**Comment by Latha2564** [comment]
![A11y_Visual Studio Code_Voiceover is not announcing the visible names for Files to Include and Exclude Fields](https://user-images.githubusercontent.com/89508906/132294313-331a6b97-f107-42e0-a7e3-876d8fddd617.GIF)

**Comment by JacksonKearl** [comment]
![recording (34)](https://user-images.githubusercontent.com/8586769/132915083-1aee2c05-3960-403b-8010-531cfb5a6f34.gif)
I found that in landmark navigation mode the headers are correctly read. This is the expected behaviour as far as I am aware, tabbing through the document normally should only read links, whereas navigating through the landmarks should read all content.

**Comment by Latha2564** [comment]
![131906](https://user-images.githubusercontent.com/89508906/133748488-c54accb9-5e2e-4668-8b06-a04624d40413.gif)

**Comment by InduPriya1805** [comment]
GitHubTags:#BM-VisualStudioCodeClient-MAC-Aug2021;
